src/main/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParser.java

```diff
@@ -947,6 +1000,7 @@ private Object constructFromString(final Class clazz, final String s, final Stri
 
     public static class ArgumentDefinition {
         public final Field field;
+        public final Class<?> type;
```

Comment:
This could be used in other parts of the code in this class instead of calling `CommandLineParser.getUnderlyingType()`. This will simplify the code a bit. Should I change it here, @cmnbroad?